src/java.desktop/macosx/native/libawt_lwawt/awt/JavaComponentAccessibility.m

```diff
@@ -1453,7 +1453,7 @@ - (id)accessibilityHitTest:(NSPoint)point withEnv:(JNIEnv *)env
     jobject jparent = fComponent;
 
     id value = nil;
-    if (JNFIsInstanceOf(env, jparent, &jc_Container)) {
+    if ((*env)->IsInstanceOf(env, jparent, jc_Container)) {
```

Comment:
Not sure, but do we need to check the jparent to NULL? "A NULL object can be cast to any class."

Reply:
Fair question but there was no NULL check there before so I assume it isn't expected to be NULL.
I could add a NULL check but then we'd skip over the following code and maybe hide something that could be a problem. I don't mind either way.

Reply:
I do not know how exactly the JNFIsInstanceOf works for NULL parameters, will it return true or false?

Reply:
I just hacked a usage in A11Y in JDK 16 :
    printf("OBJ %d\n", JNFIsInstanceOf(env, NULL, &sjc_CAccessible));

it returns TRUE for NULL - same as JNI.